M2/Macaulay2/m2/format.m2

```diff
@@ -154,6 +156,9 @@ scan({net, info},
 		BK -> ());
 	    -- Stack the pieces vertically
 	    stack x);
+	parser INDENT := x -> ( -- INDENT is like DIV but with extra |s on the left. sadly, can't be absorbed into DIV because of non-recursivity of this parser
```

Comment:
Can we use nested `DIV`s for this purpose?

Reply:
I don't think so. the way `parser` is pretty sneaky (`parser` is only called by the outermost `DIV`, after that it's all `parser'`). anyway I already spent way too much time rewriting this `INDENT` thing, this is the best I could come up with.